app/policies/work_version_policy.rb

```diff
@@ -57,7 +57,8 @@ def review?
   end
 
   def destroy?
-    (administrator? || depositor?) && record.persisted? && record.version_draft?
+    (administrator? || depositor? || reviews_collection? || manages_collection?(collection)) &&
+      record.persisted? && record.draft?
```

Comment:
note: switched `version_draft?` to `draft?` as well as adding reviewers and collection managers ... I believe this is correct because we should be able to delete a first draft of a work_version if we delete the work itself